modules/ROOT/pages/general-configuration-guide/attribution-requirements.adoc

```diff
@@ -24,7 +25,7 @@ The {productname} status bar provides the required attribution on the right-hand
 
 The following example shows the status bar product attribution in the bottom, right-hand corner:
 
-image::{baseurl}/images/tinymce5-homepage-demo.png[Screenshot of homepage {productname} demo including product attribution in the bottom, right-hand corner]
+image:tinymce5-homepage-demo.png[Screenshot of homepage {productname} demo including product attribution in the bottom, right-hand corner]
```

Comment:
Same thing as previously here, this shouldn't have been changed from a block image to an inline image.

```suggestion
image::tinymce5-homepage-demo.png[Screenshot of homepage {productname} demo including product attribution in the bottom, right-hand corner]
```